Source/WebKit/Shared/ios/WebIconUtilities.mm

```diff
@@ -104,6 +106,7 @@ static CGRect squareCropRectForSize(CGSize size)
         (id)kCGImageSourceCreateThumbnailFromImageIfAbsent: @YES,
         (id)kCGImageSourceThumbnailMaxPixelSize: @(iconSideLength),
         (id)kCGImageSourceCreateThumbnailWithTransform: @YES,
+        (id)kCGImageSourceEnableRestrictedDecoding: @YES
```

Comment:
Yes I think using `kCGImageSourceEnableRestrictedDecoding` is correct since it disables hardware decoding which may be affected by the sandbox. This is what we do when decoding images in WebCore.